cli/clierrors/pluginerror.go

```diff
@@ -0,0 +1,19 @@
+package clierrors
```

Comment:
i would not create a sepoarate package only for this error, put it into the plugins package